cmd/htcondor-api/main.go

```diff
@@ -301,6 +304,9 @@ func runDemoMode() error {
 		UIDDomain:      uidDomain,
 		Collector:      collector,
 		Logger:         logger,
+		EnableMCP:      true,                          // Enable MCP in demo mode
```

Comment:
Also enable MCP in normal mode.

Setup the DB path based on the global config.